SeaPublicWebsite.BusinessLogic/SeaPublicWebsite.BusinessLogic.csproj

```diff
@@ -3,6 +3,7 @@
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
+        <RestoreSources>$(RestoreSources);../Lib;https://api.nuget.org/v3/index.json</RestoreSources>

```

Comment:
This... Is surprising to me. I'd expect the default sources to probably include /Lib already and definitely shouldn't need to manually list nuget.
How did you come up with this? Is it a known issue with dotnet in docker?

Reply:
It's not about finding nuget, it's about finding the CabOff design package in `Lib`